glide.yaml

```diff
@@ -65,3 +65,4 @@ import:
   version: v1.1.0
 - package: github.com/onsi/ginkgo
   version: v1.3.1
+- package: github.com/duglin/vlinker
```

Comment:
this isn't a Go dependency, so I don't think it makes sense to add it as a glide dependency. it probably makes more sense to download it directly